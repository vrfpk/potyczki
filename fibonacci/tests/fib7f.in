000000032767000
