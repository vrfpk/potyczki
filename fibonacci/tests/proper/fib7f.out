1664156567375250
